CHANGELOG.md

```diff
@@ -2,10 +2,52 @@
 
 ## Table of Contents
 
+- **[r1.2](#r12)**
 - [r1.1](#r11)
 
 **Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**
 
+# r1.2
+## Release Notes
+
+This release contains the definition and documentation of
+* device-swap v0.1.0
+
+The API definition(s) are based on
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.1
+
+## device-swap v0.1.0
+
+**device-swap v0.1.0 is the 1st release of the version 0.1.0**
+
+- 0.1.0 Device Swap API definition **with inline documentation**:
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/DeviceSwap/blob/r1.2/code/API_definitions/device-swap.yaml)
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceSwap/blob/r1.2/code/API_definitions/device-swap.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceSwap/blob/r1.2/code/API_definitions/device-swap.yaml)
```

Comment:
These links won't work with the "blob"

```suggestion
  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceSwap/r1.2/code/API_definitions/device-swap.yaml&nocors)
  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceSwap/r1.2/code/API_definitions/device-swap.yaml)
```

NOTE: The same change has to be done within r1.1, the links for Redoc and Swagger there are not working.